From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: build-config / com

Files:
 build/config/win/BUILD.gn

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/build/config/win/BUILD.gn b/build/config/win/BUILD.gn
index 29b9f115f5c74..78b4a09f4f7ab 100644
--- a/build/config/win/BUILD.gn
+++ b/build/config/win/BUILD.gn
@@ -214,6 +214,10 @@ config("runtime_library") {
     "_SECURE_ATL",
   ]
 
+  # This is required for ATL to use XP-safe versions of its functions.
+  # TODO(thakis): We no longer support XP; try removing this.
+  defines += [ "_USING_V110_SDK71_" ]
+
   if (current_os == "winuwp") {
     # When targeting Windows Runtime, certain compiler/linker flags are
     # necessary.
